enhancements/network/egress-qos.md

```diff
@@ -44,6 +47,8 @@ The QoS markings will be consumed and acted upon by objects outside of the OpenS
 
 - Ingress QoS.
```

Comment:
I am guessing this needs to change because its no longer a nonGoal?

Reply:
keeping Ingress QoS as Non goal as per comment.